### DISCUSSION TITLE
Releases compared to latest main

### DISCUSSION THREAD

**Discussion by djl0** [PaulWoitaschek/Voice#1209] [General]
I'm a big fan of this app. I'm using the version from Apr 2020 from F-Droid. I see there's been a lot of development since then. Is there a plan to do another release (and post on F-Droid), or should I compile the latest APK myself if I want the latest updates?

Thanks

**Comment by PaulWoitaschek** [comment]
The current development version is not finished in terms that there are a few minor bugs and more important:
The old audiobook data is not migrated yet.

If you want to jump on the beta track, you can use the apk attached to the releases, for examlpe:
https://github.com/PaulWoitaschek/Voice/releases/tag/6.0.0-alpha15

Or you join the play beta:
https://github.com/PaulWoitaschek/Voice/wiki/FAQ#how-can-i-join-the-beta

**Comment by hurzl** [comment]
There will always be minor bugs and the release has even more.
Couldn't the release branch at least catch up somewhat? 
I can't get beta or alpha versions on F-Droid.

**Reply by PaulWoitaschek** [reply]
You can get it from the attached github releases.

**Comment by hurzl** [comment]
Yes I know
